packages/atomic/src/components/commerce/atomic-commerce-query-summary/atomic-commerce-query-summary.new.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import {wrapInCommerceInterface} from '@coveo/atomic/storybookUtils/commerce-interface-wrapper';
+import {parameters} from '@coveo/atomic/storybookUtils/common-meta-parameters';
+import {renderComponent} from '@coveo/atomic/storybookUtils/render-component';
+import {CommerceEngineConfiguration} from '@coveo/headless/commerce';
+import type {Meta, StoryObj as Story} from '@storybook/web-components';
+import {html} from 'lit/static-html.js';
+
+const {decorator, play} = wrapInCommerceInterface({
+  skipFirstSearch: true,
+});
+
+const noResultsEngineConfig: Partial<CommerceEngineConfiguration> = {
+  preprocessRequest: (r) => {
+    const parsed = JSON.parse(r.body as string);
+    parsed.query = 'NOT @URI';
```

Comment:
Hummm, this relies on query syntax, which I'm fairly certain will end up being disabled by default in commerce API.

Could we instead search for a random string that yields no result ?